examples/src/main/java/io/druid/examples/ExamplesDruidModule.java

```diff
@@ -38,9 +39,8 @@
   {
     return Arrays.<Module>asList(
         new SimpleModule("ExamplesModule")
-            .registerSubtypes(
-                new NamedType(TwitterSpritzerFirehoseFactory.class, "twitzer")
-            )
+            .registerSubtypes(new NamedType(TwitterSpritzerFirehoseFactory.class, "twitzer"))
+            .registerSubtypes(new NamedType(GuavaDoubleMath.class, "guavaDoubleMath"))
```

Comment:
i'd almost want this doubleMath extension just to be a separate extension in extensions-contrib so folks can learn to write their own math extensions rather than have the code druid-examples